composables/drop/useGenerativeDropMint.ts

```diff
@@ -0,0 +1,89 @@
+import { DoResult, DropMintedStatus } from '@/services/waifu'
+import { makeScreenshot } from '@/services/capture'
+import { pinFileToIPFS } from '@/services/nftStorage'
+
+export type DropMintedNft = DoResult & {
+  id: string
+  collectionName: string
+  name: string
+}
+
+export default ({
+  collectionData,
+  defaultMax,
+  currentAccountMintedToken,
+  collectionId,
+  mintedDropCount,
+  defaultImage,
+}: {
+  mintedDropCount: Ref<number>
+  collectionId: Ref
+  defaultImage: Ref
+  currentAccountMintedToken: Ref<DropMintedStatus | null>
+  defaultMax: Ref<number>
+  collectionData: Ref
+}) => {
```

Comment:
1. clearer if you would create a type and export it 
2. typeless `Ref`?

Reply:
done